kratos_salome_plugin/model_part.py

```diff
@@ -87,7 +87,7 @@ def __eq__(self, other):
         if not super().__eq__(other): return False
 
         if self.Id != other.Id: return False
-        if Distance(self.Coordinates(), other.Coordinates()) >  1E-15: return False
+        if Distance(self.Coordinates(), other.Coordinates()) >  1E-10: return False
```

Comment:
this is done bcs the writing precision is 10 in `WriteMdpa`

I think it is still sufficiently small